analyses/optimus/create_synthetic_data.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# this script takes as input the PBMC8k dataset, multi-aligned with Optimus.
+# it generates a bam file containing unique, unaligned, and multialigned reads from different
+# chromosomes
+# it then regenerates fastq data
```

Comment:
Your description in this PR gives more context - maybe good to copy paste some of that here.